setup.py

```diff
@@ -18,6 +18,7 @@
     setup_requires=["setuptools_scm"],
     description='Synthetic Data Generation with optional Differential Privacy',
     url='https://github.com/gretelai/gretel-synthetics',
+    license='Apache-2.0',
```

Comment:
This should be covered by the classifiers param as we are using a standard license. If pypi isn't handling that we need to figure out why. 

Ref: https://packaging.python.org/guides/distributing-packages-using-setuptools/#license

Reply:
Does this only impact "pip show?" Does installing in edit mode locally fix this when re-running the command?

Reply:
Yeah, it appears this is just for `pip` as PyPi is using the classifiers appropriately.

Reply:
It's just a `pip` vs `PyPi` thing. Pip looks for that specific field when displaying locally, but `PyPi` doesn't seem to care. Not really explained well via the Python docs IMO so I'd ship w/ this.